### PR TITLE
Update rules_docker

### PR DESCRIPTION
**Pull request by fejta**
Also vert 73f62de by reverting 934d2a1f7859987967901fab68c21855ff923b91

/cc @UebelAndre

### CODE REVIEW

**Comment by fejta** [comment]
(@UebelAndre any chance you can either respond to the `@googlebot I consent` message or else patch in the revert commit into your PR)

File: .bazelci/presubmit.yml [hunk 1]
```diff
@@ -44,15 +38,20 @@ platforms:
...
-    # TODO(nlopezgi): remove once issue above is fixed
-    - "--action_env=PATH=/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/tmp/bin/"
+  min_supported_version:
+    name: "Minimum Supported Version"
+    bazel: "2.2.0"
+    platform: ubuntu1604
+    build_targets:
+    - "--"
+    - "..."
+    - "-//images/gcloud-bazel:gcloud_push"
```
Comment (UebelAndre) [inline .bazelci/presubmit.yml:52]: I forget, why were these disabled? Seems like they should be enabled

**Comment by UebelAndre** [comment]
@googlebot I consent

File: .bazelci/presubmit.yml (hunk 1 above)
Comment (fejta) [reply .bazelci/presubmit.yml]: Probably didn't build at the time or something, let's see if they work now.

**Comment by fejta** [comment]
> @googlebot I consent

Unfortunately I think it might need to be exactly `@googlebot I consent.` (with period, which I missed initially)

**Comment by UebelAndre** [comment]
It says CLA is good to go but

@googlebot I consent.

**Comment by fejta** [comment]
oh sorry my bad!

**Comment by fejta** [comment]
(I'm used to other repos where there's a label and not just the context, thanks!)

**Comment by UebelAndre** [comment]
> (I'm used to other repos where there's a label and not just the context, thanks!)

Thanks for putting this PR together and fixing CI 😄

**Review by chases2 — APPROVED** [review]

**Pull request merged**